chipsec/cfg/8086/pch_5xxh.xml

```diff
@@ -121,18 +125,17 @@ https://cdrdv2.intel.com/v1/dl/getContent/636174
       <field name="STYPE" bit="0"  size="1"  desc="Space Type (always 0 - memory space)"/>
       <field name="BA"    bit="13" size="19" desc="Base Address"/>
     </register>
-    <register name="ABASEHIGH" type="pcicfg" device="PMC" offset="0x14" desc="Base Address High" >
+    <register name="ABASEHIGH" type="pcicfg" device="PMC" offset="0x14" size="4" desc="Base Address High" >

```

Comment:
Chipsec doesn't handle high and low memory within bars currently.  Suggest making ABASE size 8 and increasing the size of the BA.

Reply:
Agreed.